romana-install/roles/stack/devstack/prep/tasks/disable_virbr0.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Disable libvirt from being started on installation
```

Comment:
A comment about why we are installing it in the first place, if we immediately stop it from running.